.circleci/config.yml

```diff
@@ -17,6 +17,7 @@ workflows:
   dist-compile:
     jobs:
       - linux-build
+      - linux-build-release
```

Comment:
I'll remove this - just wanted to trigger it once from this PR to ensure the jobs goes through.